site/docs/expressions/scalar_functions.md

```diff
@@ -1,52 +1,118 @@
 # Scalar Functions
 
-Scalar functions are a function that takes in values from a single record and produces an output value. To clearly specify the definition of functions, Substrait declares a extensible specification plus binding approach to function resolution. This ensures a clear definition of function behavior separate from a particular use of each function while avoiding operations 
+Scalar functions are a function that takes in values from a single record and produces an output value. To clearly specify the definition of functions, Substrait declares a extensible specification plus binding approach to function resolution. A scalar function signature includes the following properties:
```

Comment:
```suggestion
A function is a scalar function if that function takes in values from a single record and produces an output value. To clearly specify the definition of functions, Substrait declares a extensible specification plus binding approach to function resolution. A scalar function signature includes the following properties:
```
You could also maybe do "Scalar functions are functions that..."  but "Scalar functions are a function that" sounds weird to me.

This paragraph itself is rather intimidating and unclear to a neophyte but we tackle that more in the post-consensus phase.